faq/index.md

```diff
@@ -2,153 +2,280 @@
 
-## <a name="router"></a>What is the difference between a router and an antenna?
+## <a name="businessplan"></a>What is your business plan?
+We are not a business and we are not a commercial Internet Service Provider! We aim to grow our community in a sustainable and inclusive way for the benefit of all of our members. We don’t see members as customers, consumers or users, but as contributors to our community. Being a member of our network has many benefits, including replacing your current ISP with fast, neutral, affordable, and unmonitored Internet access. Businesses are free to join or connect to the mesh as long as they obey the [network commons license](../ncl.pdf).
 
-Most of the devices we use, such as a LiteBeam or NanoStation are self-contained so they have an antenna, radio and ethernet router all in one. "Wireless router with antenna" is too long so we usually call the whole thing a router, just like your home wifi "router". Sector antennas usually have a separate radio plugged into the back of them so we call those antennas (although some sectors are self-contained). 
+## <a name="meshonly"></a>What can I do on the mesh that I can’t do on the internet?
 
-Ubiquiti call their self-contained AirFiber a "radio", and others call an AirFiber a router or an antenna, so yes everyone is confused about what term to use.
+Other than the benefit of not paying the cable cartel for overpriced Internet access, it is also possible to connect directly to other people on the mesh without using the Internet. If for some reason the Internet is shut down (hurricane, government crackdown, cyberattack, zombie apocalypse), the mesh will likely still work for a while longer. 
 
-## <a name="firmware"></a>What software/firmware do you use?
+The mesh also has its own web servers that show local information and allow you to chat and post messages. So far we have servers in five locations: Harlem, Bay Ridge, 3rd St. and 7th St. in the East Village, and our main directory “wiki.mesh” in Williamsburg. We are in need of [volunteers](../help) to help add content and maintain our servers.
 
-We are using an OpenWrt package made by qMp. This uses the BMX6 mesh protocol. We have added tinc tunneling so a router can mesh over the internet when out of range using a secure virtual private network (VPN). 
+# Joining the Community
 
-Our point-to-point connections use factory firmware such as AirOS. The supernode is running Linux with Quagga and BGP. For network monitoring we use Nagios and Icinga2.
+## <a name="signup"></a>How do I sign up?
 
-We also have some members experimenting with and contributing to cjdns/hyperboria software. Let us know if you are interested in this.
+After reading through the FAQs, fill out the [join form](../join) to be assigned a node, and look for an email from one of our volunteers with next steps (check your email spam folder or contact us directly at install@nycmesh.net if you don’t receive any reply).
```

Comment:
linkify email address to a mailto:

Reply:
I feel like many people don't have an email client set up for mailto to point towards - I think it's easier to just copy and paste email address.